### PR TITLE
Make namespace configurable (part 2)

### PR DESCRIPTION
**Pull request by mrylov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html for detai
 
 - Create a build directory and change to it:
     ```
-    mkdir odbc-cpp-wrapper\build && cd odbc-cpp-wrapper\build
```
Comment (stefanuhrig) [inline README.md:84]: Why did you change from backslash to slash?

Comment (mrylov) [reply README.md:84]: I was resolving the doxygen's warnings and forgot to rollback the change in this file.
